### PR TITLE
Add common default MIDI controllers

### PR DESCRIPTION
**Pull request by gwilherm**
Added the following MIDI CC as default mapping:
```
MIDI CC 5  :  kAmsynthParameter_PortamentoTime
MIDI CC 71 :  kAmsynthParameter_FilterResonance
MIDI CC 72 :  kAmsynthParameter_AmpEnvRelease
MIDI CC 73 :  kAmsynthParameter_AmpEnvAttack
MIDI CC 74 :  kAmsynthParameter_FilterCutoff
MIDI CC 91 :  kAmsynthParameter_ReverbWet
```

### CODE REVIEW

**Review by nickdowell — CHANGES_REQUESTED** [review]
Thank you for this contribution!

I agree it's a sensible use of the standard MIDI CCs, just a few minor suggested changes from me.

File: src/core/midi.h [hunk 1]
```diff
@@ -48,6 +50,11 @@ enum {
     MIDI_CC_SUSTAIN_PEDAL               = 0x40,
     MIDI_CC_PORTAMENTO                  = 0x41,
     MIDI_CC_SOSTENUTO                   = 0x42,
+    MIDI_CC_FILTER_RESONANCE            = 0x47,
+    MIDI_CC_AMP_ENV_RELEASE             = 0x48,
+    MIDI_CC_AMP_ENV_ATTACK              = 0x49,
+    MIDI_CC_FILTER_CUTOFF_FREQ          = 0x4A,
+    MIDI_CC_REVERB_LEVEL                = 0x5B,
```
[the remark this answers is not in the document]
Comment (gwilherm) [reply src/core/midi.h:57]: You're right

File: src/core/synth/MidiController.cpp [hunk 2]
```diff
@@ -211,7 +211,16 @@ MidiController::controller_change(unsigned char cc, unsigned char value)
...
+			break;
 		case MIDI_CC_MODULATION_WHEEL_MSB:
+		case MIDI_CC_PORTAMENTO_TIME:
+		case MIDI_CC_VOLUME:
+		case MIDI_CC_FILTER_RESONANCE:
+		case MIDI_CC_FILTER_CUTOFF_FREQ:
+		case MIDI_CC_AMP_ENV_ATTACK:
+		case MIDI_CC_AMP_ENV_RELEASE:
+		case MIDI_CC_REVERB_LEVEL:
+			/* Handled by controller map */
```
[the remark this answers is not in the document]
Comment (gwilherm) [reply src/core/synth/MidiController.cpp:223]: No they are not necessary of course, I've been maybe kinda influenced by the presence of `MIDI_CC_MODULATION_WHEEL_MSB` case

[the remark this answers is not in the document]
Comment (nickdowell) [reply src/core/synth/MidiController.cpp]: Actually these lines might help to silence any compiler / lint warnings about unhandled cases 🙃 let's keep them

**Review by nickdowell — APPROVED** [review]

**Comment by gwilherm** [comment]
Thanks for the merge and for your awesome work !

**Pull request merged**